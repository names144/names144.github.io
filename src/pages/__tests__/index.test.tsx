import * as React from 'react';
import renderer from 'react-test-renderer';

import Index from '../index';

jest.mock('components/Seo');

describe('Index', () => {
  test('Should render', () => {
    const tree = renderer
      .create(<Index />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
