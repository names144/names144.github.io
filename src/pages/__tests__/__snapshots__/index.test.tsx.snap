// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Should render 1`] = `
Array [
  <mock-seo />,
  <div
    className="md:container md:mx-auto"
  >
    <span
      className="text-xl"
    >
      TODO
    </span>
  </div>,
]
`;
